### DISCUSSION TITLE
function balanceOf

### DISCUSSION THREAD

**Discussion by veridelisi** [freitasgouvea/token-erc-20#2] [General]
Dear 
I dont understand why you used "**balance** " keyword in returns. What is the role ?

 function balanceOf(address _owner) public override view returns (uint256 **balance** ) 
    {
        return _balances[_owner];
    }

**Comment by freitasgouvea** [comment]
Hi @veridelisi in this case it has no effect, since I use the return statement inside function. Maybe I just wanted to give a name because it is possible, I don't remember.

**Reply by veridelisi** [reply]
Obrigado
Saludos desde Turquia
